path_config

```diff
@@ -16,6 +16,11 @@ path /status StatusHandler {
     root .;
 }
 
+path /reverse_proxy ReverseProxyHandler {
+    remote_host localhost;
+    remote_port 1234;
```

Comment:
I'm confused about this. Do you mind expanding on it?
The assignment says we have to connect to and display the UCLA website.